src/Uno.UWP/Media/Capture/CameraCaptureUI.iOS.cs

```diff
@@ -0,0 +1,206 @@
+#if __IOS__
```

Comment:
Same

Reply:
Uno.UI has no `.iOS` implicit globbing for now.

Reply:
Uno has globbing enabled, removing those lines.